### PR TITLE
DOC-5085: 4xx Page aliases for query

### PR DESCRIPTION
**Pull request by simon-dew**
- [x] https://docs.couchbase.com/server/4.6/n1ql/n1ql-language-reference/string-functions.html
- [x] https://docs.couchbase.com/server/6.0/n1ql/n1ql-intro/n1ql-sql-differences.html
- [x] https://docs.couchbase.com/server/6.0/rest-api/rest-bucket-auth.html
- [x] https://docs.couchbase.com/server/6.0/rest-api/rest-buckets-streaminguri.html
- [x] https://docs.couchbase.com/server/6.0/developer-guide/query-workbench-intro.html

### CODE REVIEW

File: modules/n1ql/pages/n1ql-language-reference/stringfun.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 = String Functions
 :page-topic-type: concept
+:page-aliases: n1ql:n1ql-language-reference/string-functions
```
Comment (eric-schneider) [inline modules/n1ql/pages/n1ql-language-reference/stringfun.adoc]: Doesn't this only need to be fixed in the 4.6 branch? https://docs.google.com/spreadsheets/d/1Kr_qOWdBd-12ak-HenRaX6-Z4X3rqNPk3B9CI-my3Ps/edit#gid=24988422&range=H1038

File: modules/rest-api/pages/rest-buckets-streamingURI.adoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 = Getting Bucket Streaming URI
 :page-topic-type: reference
+:page-aliases: rest-buckets-streaminguri
```
Comment (eric-schneider) [inline modules/rest-api/pages/rest-buckets-streamingURI.adoc:3]: This one seems to be correcting for the fact that the old site didn't support case-sensitive URLS?

File: modules/tools/pages/query-workbench.adoc [hunk 3]
```diff
@@ -1,4 +1,5 @@
 = Query Workbench
+:page-aliases: developer-guide:query-workbench-intro
```
Comment (eric-schneider) [inline modules/tools/pages/query-workbench.adoc:2]: This needs to be fixed in 4.1 as well. I assume that a PR for 4.1 is coming?

**Review by eric-schneider — CHANGES_REQUESTED** [review]
These look good. I'm just concerned about the first one. The page alias might be reserved somewhere else in the 6.0 branch for n1ql:n1ql-intro/n1ql-sql-differences since it doesn't appear in the spreadsheet. Or, something out there is linking to specifically the 4.6 branch version of that page.

File: modules/rest-api/pages/rest-buckets-streamingURI.adoc (hunk 2 above)
Comment (simon-dew) [reply modules/rest-api/pages/rest-buckets-streamingURI.adoc:3]: Right, it's just correcting for the lower-case uri. But it needs fixing with the current site, apparently.

File: modules/n1ql/pages/n1ql-language-reference/stringfun.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/stringfun.adoc]: Created #484 for release/4.6

Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/stringfun.adoc]: Removed redirect in the 6.0 branch

Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/stringfun.adoc]: Wait a second, it's back

Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/stringfun.adoc]: Okay, removed redirect in the 6.0 branch again. Some kind of merge conflict there.

**Review by eric-schneider — APPROVED** [review]

File: modules/tools/pages/query-workbench.adoc (hunk 3 above)
Comment (simon-dew) [reply modules/tools/pages/query-workbench.adoc]: Created #485 for release/4.1

**Comment by simon-dew** [comment]
Redirect removed from rest-bucket-auth because of alias conflict, see #488

**Pull request merged**